data.js

```diff
@@ -77,6 +77,7 @@ const pages = [
     description:
       'Independent Software Engineer, trainer & speaker. Into sports & music.',
     url: 'https://glennreyes.com/uses',
+    twitter: '@glnnrys',
```

Comment:
@wesbos idk why this addition is here (maybe due to other PR's being merged in before I created mine. I left it here though thinking you would have more context to know what to do with it.